cebra/solver/multiobjective.py

```diff
@@ -445,7 +457,7 @@ def validation(
         return stats_val
 
     @torch.no_grad()
-    def transform(self, inputs: torch.Tensor) -> torch.Tensor:
+    def transform_deprecated(self, inputs: torch.Tensor) -> torch.Tensor:
```

Comment:
this needs a longer comment on the rationale, I think; plus the function should raise some kind of error message or warning when called